backend/analyzer.py

```diff
@@ -27,11 +34,184 @@ def from_friendly_name(name):
         'tram': T.tramway,
         }
 
+Transportation = collections.namedtuple('Transportation', 'type line')
 
 
-line_number_pattern = re.compile(r'^([0-9]{1,3}|[a-z]{1,2})$', re.I)
 
-Transportation = collections.namedtuple('Transportation', 'type line')
+################################################
+# Normalization
+
+class shortcuts:
+    """namespace"""
+    remove = lambda L, x: list(filter(lambda i:i not in x, L))
```

Comment:
You should add a space after `lambda i:` like you did with others lambda declarations, or is there a reason?